.github/workflows/cypress-custom.yml

```
@@ -92,12 +92,12 @@ jobs:
         with:
           path: apps/${{ env.APP_NAME }}
 
-      - name: Set up node 16
+      - name: Set up node 18
         uses: actions/setup-node@v3
         with:
           cache: 'npm'
           cache-dependency-path: apps/${{ env.APP_NAME}}/package-lock.json
-          node-version: 16
+          node-version: 18
```

if specified in package.json under `engine` you can also use:
```suggestion
          node-version-file: package.json
```